ui/app/styles/core/progress.scss

```diff
@@ -37,3 +45,41 @@
   border-radius: $radius;
   transition: width 1s ease-out;
 }
+
+.progress::-ms-fill {
+  background-color: #4a4a4a;
+  border: none;
+}
+
+// is-success and is-info are the two types of progress bars we have
+.progress.is-info::-webkit-progress-value {
+  background-color: #3e8ed0;
```

Comment:
should these hex codes be a variable? I see them used it a few files

Reply:
Good spot. Yes, eventually. I've got a whole ticket to address the 500 shades of colors we have running around. For now, I'm just copy pasting from bulma and will address colors all at once.